src/components/sidebar/SidebarToc.svelte

```diff
@@ -10,23 +10,30 @@
         hidden?: boolean;
     }[];
 
+    export let minHeadingDepth: number = 1;
     export let maxHeadingDepth: number = 4;
 
+    // filter out headings that are lower than min_heading_depth
+    headings = headings.filter((h) => h.depth >= minHeadingDepth);
+
     // filter out headings that are higher than max_heading_depth
     headings = headings.filter((h) => h.depth <= maxHeadingDepth);
 
-    const min_heading_depth = Math.min(...headings.map((h) => h.depth));
+    minHeadingDepth = Math.min(...headings.map((h) => h.depth));
```

Comment:
This now overwrites the supplied config, I think? Needs a different variable name and then a second `max` call on the two..

Reply:
No, it should be fine. Because the supplied config is already filtering out higher level headings, so the `min` hear can't be lower then minHeadingDepth.

Reply:
Right, yup 👍🏻